src/directives/gmf-map.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @fileoverview This file provides the "map" directive
+ * for GeoMapFish applications.
+ *
+ * Example:
+ *
+ * <ngeo-gmf-map ngeo-gmf-map-map="::mainCtrl.map"></ngeo-gmf-map>
+ */
+goog.provide('ngeo.gmf.mapDirective');
+
+goog.require('goog.asserts');
+goog.require('ngeo.Debounce');
+goog.require('ngeo.Location');
+goog.require('ngeo.mapDirective');
+
+
+/**
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+ngeo.gmf.mapDirective = function() {
+  return {
+    scope: {},
+    bindToController: {
```

Comment:
@elemoine maybe to put parent scope properties in your directive controller ? You can still want a `scope:true` but use the binding for you controller through dom attributes. I don't really know what could be the usecase.